app/views/travel_advice/index.html.erb

```diff
@@ -13,6 +13,14 @@
       <%= render "govuk_publishing_components/components/lead_paragraph", {
         text: @presenter.description,
       } %>
+      <div class="govuk-grid-row travel-advice__header">
+        <div class="govuk-grid-column-two-thirds">
+          <%= render "govuk_publishing_components/components/warning_text", {
+            text: "Text <a href=\"https://gov.uk/\" class=\"govuk-link\">Link</a>".html_safe,
```

Comment:
Whoever updates with the final content, can I suggest: `text: sanitize("Text #{link_to('Link', '/url', class: "govuk-link")}")` instead of escapes and _html_safe_

Reply:
A much better way! Thanks @alex-ju 😄